daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/dsom/SDE.scala

```diff
@@ -49,6 +49,26 @@ class SchemaDefinitionError(
 
 }
 
+class SchemaDefinitionErrorFromWarning(
+  sdw: SchemaDefinitionDiagnosticBase,
+  warnID: WarnID
+) extends SchemaDefinitionWarning(
+    warnID,
+    sdw.sc.toScalaOption,
+    sdw.annotationContext.toScalaOption,
+    ({
+      val origMsg = if (sdw.mfmt.isDefined) sdw.mfmt.get else ""
+      val msg = s"warning escalated to error: ${warnID}. " + origMsg
```

Comment:
I think the modeName is already "Schema Definition Warning Escalated Error". Could we remove the duplicate "warning escalated to error" in the message?

Reply:
+1